tables/output_tables.Rmd

```diff
@@ -13,132 +12,168 @@ library(openxlsx)
                      )
-openxlsx::write.xlsx(list_s2_table, 
-                     file.path(results_dir, "TableS2-DNA-results-table.xlsx"),
-                     overwrite=TRUE)
+write.xlsx(list_s2_table, 
+           table_s2_file,
+           overwrite=TRUE)
 ```
 
 ## Table S3: RNA results table
 ### TP53 scores
 ```{r s3 RNA results table p1}
 # get tp53 scores 
-tp53_scores <- readr::read_tsv("../analyses/tp53_nf1_score/results/tp53_altered_status.tsv") %>%
+tp53_scores <- read_tsv(tp53_file) %>%
   mutate_all(as.character) %>% 
   replace(is.na(.), "NA")
 ```
 
 ### Telomerase scores
 ```{r s3 RNA results table p2}
 # get extend scores file
-telomerase_scores_polya_count <- readr::read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAPolya_counts.txt") %>%
-  dplyr::select(SampleID, NormEXTENDScores) %>% 
-  dplyr::rename(Kids_First_Biospecimen_ID_RNA = SampleID,
+telomerase_scores_polya_count <-
+  read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAPolya_counts.txt") %>%
+  select(SampleID, NormEXTENDScores) %>% 
+  rename(Kids_First_Biospecimen_ID_RNA = SampleID,
                 NormEXTENDScores_counts = NormEXTENDScores)
 
 telomerase_scores_polya_fpkm <-
-  readr::read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAPolya_FPKM.txt") %>%
-  dplyr::select(SampleID, NormEXTENDScores) %>% 
-  dplyr::rename(Kids_First_Biospecimen_ID_RNA = SampleID,
+  read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAPolya_FPKM.txt") %>%
+  select(SampleID, NormEXTENDScores) %>% 
+  rename(Kids_First_Biospecimen_ID_RNA = SampleID,
                 NormEXTENDScores_fpkm = NormEXTENDScores)
 
 telomerase_scores_polya_combined <- full_join(telomerase_scores_polya_count,
                                               telomerase_scores_polya_fpkm)
   
-telomerase_scores_stranded_count <- readr::read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAStranded_counts.txt") %>%
-  dplyr::select(SampleID, NormEXTENDScores) %>% 
-  dplyr::rename(Kids_First_Biospecimen_ID_RNA = SampleID,
+telomerase_scores_stranded_count <-
+  read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAStranded_counts.txt") %>%
+  select(SampleID, NormEXTENDScores) %>% 
+  rename(Kids_First_Biospecimen_ID_RNA = SampleID,
                 NormEXTENDScores_counts = NormEXTENDScores)
 
-telomerase_scores_stranded_fpkm <- readr::read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAStranded_FPKM.txt") %>%
-  dplyr::select(SampleID, NormEXTENDScores) %>% 
-  dplyr::rename(Kids_First_Biospecimen_ID_RNA = SampleID,
+telomerase_scores_stranded_fpkm <-
+  read_tsv("../analyses/telomerase-activity-prediction/results/TelomeraseScores_PTBAStranded_FPKM.txt") %>%
+  select(SampleID, NormEXTENDScores) %>% 
+  rename(Kids_First_Biospecimen_ID_RNA = SampleID,
```

Comment:
Can't easily comment on this in this diff - do we want to add a column that indicates which data set these samples come from (`stranded`, `polya`)?

Reply:
Realizing I missed all of that telomerase jazz on Friday afternoon. Library column has been added!